extensions/hazelcast/deployment/src/main/java/org/apache/camel/quarkus/component/hazelcast/deployment/HazelcastProcessor.java

```diff
@@ -51,6 +52,10 @@
 
     private static final String FEATURE = "camel-hazelcast";
 
+    private static final String[] RUNTIME_INITIALIZED_CLASSES = new String[] {
+            "com.hazelcast.internal.util.ICMPHelper"
```

Comment:
Small nit - If there's just one class, do we need an array?

Reply:
Hi @jamesnetherton ,

I can use a String instead, but put a String Array here just in case this list may get expanded.
Thanks!
Freeman